src/corehost/packaging/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I didn't do that because you only have all your projects in the root of that directory. I will have a look and restructure this.